src/components/SideBar.tsx

```diff
@@ -84,44 +87,123 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const SideBar = (props: SideBarProps) => {
+  dotenv.config();
   const classes = useStyles();
   const theme = useTheme();
+
+  const [userProfileTemplateId, setUserProfileTemplateId] = useState<any>('');
+  const [listOfApps, setListOfApps] = useState<any>([]);
+  const [loadedData, setLoadedData] = useState<any>(false);
+
+  const appsData: any = [...listOfApps];
+
+  const sendUrl = (url: string) => {
+    return request(url);
+  };
+
+  const getProfileTemplateAndAppIds = () => {
+    let baseUrl = `/api/v1/user/types`;
+    axios
+      .all([
+        sendUrl(baseUrl),
+        sendUrl(`/api/v1/apps/user/types?expand=app%2CappLogo&category=apps`),
+      ])
+      .then(
+        axios.spread((...responses) => {
+          let responseOne = responses[0];
+          let responseTwo = responses[1];
+          let defaultId = responseOne!.data[0].id;
+          let appData = responseTwo!.data;
```

Comment:
I think const would be a better declaration for the above variables @SaiSrinivasPendyala

Reply:
Done, changed to const.